### DISCUSSION TITLE
Realtime update not returning payload.new

### DISCUSSION THREAD

**Discussion by simondpalmer** [supabase/supabase#15879] [Questions]
I have subscribed to my table and receive a notification of an update made but I cant access payload.new?
My payload contains errors: ["Error 401"]. See below payload callback:
```
{
    "schema": "public",
    "table": "LotInterest",
    "commit_timestamp": null,
    "eventType": "UPDATE",
    "new": {},
    "old": {},
    "errors": [
        "Error 401: Unauthorized"
    ]
}
```
I have enabled Realtime for table "LotInterest" in the dashboard and have created a policy. see below:
```
CREATE POLICY "policy_name"
ON public.LotInterest
FOR SELECT USING (
  true
);
``` 
But still I cant access payload.new
UPDATE:
Below is my subscription:
```
useEffect(() => {
      let interestId = lotInterest.id
      supabase
        .channel("any")
        .on(
          "postgres_changes",
          { event: "*", schema: "*"},
          (payload) => {
            console.log("New Payload", payload.new);
          }
        )
        .subscribe();
    }, []);
```
Am I missing something?

**Comment by GaryAustin1** [comment]
Did your policy SQL run without error? You have capital letters in your table name, which is a pain in Postgres.  Your policy would need to use `public."LotInterest"` to run without error, IF your table name is really capitalized.  Double quotes have to be used around capital letters in Postgres names.

Note though I'm not sure you would get any response at all if you did not meet RLS.
